### PR TITLE
[TASK] Add Rails 5.0 to the build matrix

### PR DESCRIPTION
**Pull request by oliverklee**
Also update Ruby 2.2 to 2.2.5 and 2.3.0 to 2.3.1 for the Travis builds.

### CODE REVIEW

File: gemfiles/rails_5.0.gemfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# This file was generated by Appraisal
+
+source "http://rubygems.org"
+
+gem "rails", "~>4.2.0"
```
Comment (lwe) [inline gemfiles/rails_5.0.gemfile]: shouldn't this be `~> 5.0.0`?

**Comment by lwe** [comment]
Thanks

File: gemfiles/rails_5.0.gemfile (hunk 1 above)
Comment (oliverklee) [inline gemfiles/rails_5.0.gemfile]: Thanks, fixed and re-pushed.

**Comment by oliverklee** [comment]
Rebased.

**Comment by lwe** [comment]
Thanks for the rebase 👌

**Pull request merged**